### ISSUE TITLE
Error importing BAM file

### ISSUE THREAD

**Issue by RantonioR** [GreenleafLab/ArchR#1320]
Hello everyone,

I've been having problems creating the ArrowFiles from my bam file. From the log file, it seems that the function works fine until it reaches the step of "Adding Chromosomes".

Log File:

[ArchR-createArrows-c02c415a83a3-Date-2022-03-03_Time-20-20-55.log](https://github.com/GreenleafLab/ArchR/files/8181332/ArchR-createArrows-c02c415a83a3-Date-2022-03-03_Time-20-20-55.log)

This is the script I'm running:

```
library(ArchR)
library(parallel)
library(magick)
set.seed(123)

addArchRThreads(threads = 8)
inputFiles <- "210526_NB501183.sorted.bam"
addArchRGenome("hg38")

ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = "210526_NB501183",
  minTSS = 4, 
  minFrags = 1000, 
  addTileMat = TRUE,
  bcTag = 'RG',
  addGeneScoreMat = TRUE
)
```

And these are four entries of the bam file:

> NB501183:919:HT52YAFX2:2:11202:8199:9105        147     chr1    564912  255     75M     =       564709  -278    ATCCAAAATTCTCCGTGCCACCTATCACACCCCATCCTAAAGTAAGGTCAGCTAAATAAGCTATCGGGCCCATAC     EEEAAEEEEEEAEEEEEEEEEEE6EEEEEEEEEEEEEEEEEEEEEAAEEEEEEEEEAEEEEEEEEEEEEEAAAAA     AS:i:0  XN:i:0  XM:i:0  XO:i:0  XG:i:0  NM:i:0  MD:Z:75 YS:i:0  YT:Z:CP RG:Z:B_1_1_S97

> NB501183:919:HT52YAFX2:2:21306:21829:15394      83      chr1    564912  255     75M     =       564601  -386    ATCCAAAATTCTCCGTGCCACCTATCACACCCCATCCTAAAGTAAGGTCAGCTAAATAAGCTATCGGGCCCATAC     EEE/<A<AEEEEEEAAEAEEEE/EEEEEEEEEEEEAEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEAEEEAAAAA     AS:i:0  XN:i:0  XM:i:0  XO:i:0  XG:i:0  NM:i:0  MD:Z:75 YS:i:-8 YT:Z:CP RG:Z:B_1_1_S97

> NB501183:919:HT52YAFX2:3:21412:12788:11277      83      chr1    564912  255     75M     =       564902  -85     ATCCAAAATTCTCCGTGCCACCTATCACACCCCATCCTAAAGTAAGGTCAGCTAAATAAGCTATCGGGCCCATAC     EEAEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEEEEAAAAA     AS:i:0  XN:i:0  XM:i:0  XO:i:0  XG:i:0  NM:i:0  MD:Z:75 YS:i:0  YT:Z:CP RG:Z:B_1_1_S97

> NB501183:919:HT52YAFX2:3:21608:18979:20255      83      chr1    564912  255     75M     =       564771  -216    ATCCAAAATTCTCCGTGCCACCTATCACACCCCGTCCTAAAGTAAGGTCAGCTAAATAAGCTATCGGGCCCATAC     EE/EEEEEEEEEEEEEEEEAEAEEEEEEAEEEEEEEEEEEEEEEEEE6EEEEEEEEEEEEEEEEEEEEEEAAAAA     AS:i:-5 XN:i:0  XM:i:1  XO:i:0  XG:i:0  NM:i:1  MD:Z:33A41      YS:i:0  YT:Z:CP RG:Z:B_1_1_S97

I've tried with different bam files and reducing the number of threads to 1, but I always run into the same problem. Any suggestion is appreciated.

Thanks!!

**Comment by rcorces** [comment]
Hi @RantonioR! Thanks for using ArchR! Please make sure that your post belongs in the Issues section. Only bugs and error reports belong in the Issues section. Usage questions and feature requests should be posted in the [Discussions](https://github.com/GreenleafLab/ArchR/discussions) section, not in Issues.<br />
__Before we help you, you must respond to the following questions__ unless your original post already contained this information:
__1.__ If you've encountered an error, have you already searched previous Issues to make sure that this hasn't already been solved?
__2.__ Can you recapitulate your error using the tutorial code and dataset? If so, provide a reproducible example.
__3.__ Did you post your log file? If not, add it now.

**Comment by rcorces** [comment]
We dont work with bam file input too often so I'm not super familiar with everything that can go wrong. Can you follow the relevant code in https://github.com/GreenleafLab/ArchR/issues/589 to download a 10x Genomics publicly-available BAM file and see if you can create arrows from that?

One thing to check, which is shown in that issue, is that if some of your fragments dont have a value in the RG tag, that will break Arrow creation.

**Comment by RantonioR** [comment]
Hi Ryan,

Thanks for your suggestions. I tried creating the ArrowFiles following the code from #589 and it seems the process halts at the same step. See [LogFile](https://github.com/GreenleafLab/ArchR/files/8190879/ArchR-createArrows-c02c3315049c-Date-2022-03-05_Time-14-50-15.log). Before uploading the file I filtered out the fragments without RG tags as suggested in #589 so I don't think that is the problem.

I am going to try converting my bam files to fragment files using [Sinto](https://timoast.github.io/sinto/basic_usage.html#create-scatac-seq-fragments-file). Let's see if that helps!
